profile_library/reolink/CX410W (W320X)/model.json

```
@@ -0,0 +1,13 @@
+{
+  "author": "Robert M. Albrecht <powercalc@romal.org>",
+  "calculation_strategy": "fixed",
+  "created_at": "2025-12-14T13:00:00",
+  "device_type": "camera",
+  "fixed_config": {
+    "power": 8.5
+  },
+  "measure_description": "Manually measured",
+  "measure_device": "Nous A1T",
+  "measure_method": "manual",
+  "name": "CX410W (W320X)"
```

You can put the marketed product name here.
It is used in the library website.

```suggestion
  "name": "ColorX Smart 2K Wi-Fi Security Camera with True Full-Color Night Vision"
```